aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -77,6 +77,35 @@ module aptos_std::simple_map {
         vector::push_back(&mut map.data, Element { key, value });
     }
 
+    public fun to_vec<Key: store, Value: store>(
```

Comment:
What is the use case for this? This seems rather expensive + hard to use as you get a vector of Elements back and need to call split_element on each of them

Reply:
Relics from destroy